packages/core/src/components/Button/Button.tsx

```diff
@@ -106,10 +109,9 @@ export const HvButton: <C extends React.ElementType = "button">(
   ) => {
     const {
```

Comment:
@zettca can we remove the default value of `base` on the ` radius` prop? I think this is still ignoring the `theme.button.borderRadius` prop, preventing us from using that theme override.

Reply:
yeah, makes sense 👍